ansible-playbooks/ros_install_and_set_up.yml

```diff
@@ -9,7 +9,7 @@
   tasks:
     - name: Enable required repository
       rhsm_repository:
-        name: rhel-8-for-x86_64-baseos-rpms,rhel-8-for-x86_64-appstream-rpms
+        name: rhel-8-for-x86_64-baseos-rpms,rhel-8-for-x86_64-appstream-rpms,rhel-8-for-aarch64-baseos-rpms,rhel-8-for-aarch64-appstream-rpms
```

Comment:
Will it be possible to make use of ansible facts?

Reply:
done

Reply:
Tested on ARM. Works as expected.

Reply:
Could you please also attach some logs for reference?

Reply:
Attached screenshot at top.